test/runtests.jl

```diff
@@ -1,20 +1,26 @@
 using Convex
 using Convex.ProblemDepot: run_tests
 using Test
-using SCS, ECOS, GLPKMathProgInterface
+using SCS, ECOS, GLPK
 
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIU = MOI.Utilities
 
 # Seed random number stream to improve test reliability
 using Random
 Random.seed!(2)
 
 @testset "ProblemDepot" begin
-    @testset "Problems can run without `solve!`ing if `test==false`" begin
+    @testset "Problems can run without `solve!`ing if `test==false`; T=$T" for T in (Float64, BigFloat)
```

Comment:
We check that our tests run even when the problem is specified with high precision numeric types